star.wdl

```diff
@@ -12,6 +12,7 @@ task Star {
     String? outStd
     String? twopassMode
     Array[String]? outSAMattrRGline
+    Int? limitBAMsortRAM
```

Comment:
Just checking: Did you make sure does not also accept floats?

Reply:
The STAR manual says it should be an int.

Reply:
:+1: